cargo-dist/templates/ci/github/release.yml.j2

```diff
@@ -402,12 +409,30 @@ jobs:
           echo "EOF" >> "$GITHUB_OUTPUT"
 
           cp dist-manifest.json "$BUILD_MANIFEST_NAME"
+      {{%- if need_cargo_cyclonedx %}}
+      - id: cargo-cyclonedx
+        shell: bash
+        run: |
+            # Generate SBOM (.cdx.xml) files.
+            cargo cyclonedx -v
```

Comment:
At some point, it may be useful for us to move this into Rust so that we can also run this locally as a part of global builds. I don't think that's a priority for us right now, given we don't install it for users outside of CI and most people are going to be doing this in our actions runs.

Reply:
It might make sense to ask the `cargo-cyclonedx` folks about an option to dump all the generated files in a specific directory. That'd _greatly_ simplify doing this from Rust, because we could avoid having to move files around after running it.